src/components/playback/playbackmanager.js

```diff
@@ -1848,6 +1848,15 @@ class PlaybackManager {
                     SortBy: options.shuffle ? 'Random' : 'SortName',
                     MediaTypes: 'Audio'
                 });
+            } else if (firstItem.Type === 'Series') {
```

Comment:
https://github.com/jellyfin/jellyfin-web/pull/4827#discussion_r1348019090
> Ah thanks, though when it comes to `'Season'` the interaction has to be different since one should queue all episodes after the present one. A separate future PR makes sense methinks.

This logic can be implemented like:
```js
// Use SeriesId if exists
promise = apiClient.getEpisodes(firstItem.SeriesId || firstItem.Id, {
...
// Skip other seasons
const isSeries = firstItem.Type === 'Series';
...
if (!e.UserData.Played
    && (isSeries || e.SeasonId === firstItem.Id)) {
    foundItem = true;
    return true;
}
```
But it doesn't work if the season is complete (watched) - no items.
We probably should restart it.

Also, maybe for seasons, only the selected one should be played. :thinking: